### PR TITLE
Implement navbar

### PR DESCRIPTION
**Pull request by joedixon**
Screenshot 2019-06-27 at 21 19 07

Screenshot 2019-06-27 at 21 23 28

Screenshot 2019-06-27 at 21 19 32

Screenshot 2019-06-27 at 21 19 44

Screenshot 2019-06-27 at 21 20 00

### CODE REVIEW

File: resources/css/app.css [hunk 1]
```diff
@@ -1,5 +1,118 @@
...
+nav ul li:last-child {
+    @apply pb-0;
+}
+
+nav ul.subnav li.divider {
+    @apply hidden;
+}
+
+@screen lg {
+
```
Comment (driesvints) [inline resources/css/app.css]:
```suggestion
```

File: resources/js/app.js [hunk 2]
```diff
@@ -16,6 +16,33 @@ require('./bootstrap');
 $('select.selectize').selectize({ maxItems: 3 });
 $('textarea.wysiwyg').markdown({ iconlibrary: 'fa' });
 
+$('.dropdown').on('click', function(e) {
+    e.preventDefault();
+    e.stopPropagation();
+    const activate = $(this).next().hasClass('active') ? false : true;
+    $('.subnav').removeClass('active');
+    if(activate) {
```
Comment (driesvints) [inline resources/js/app.js]: Also add some white lines here to let the code breath a little.

```suggestion
    if (activate) {
```

File: resources/views/layouts/_nav.blade.php [hunk 3]
```diff
@@ -1,31 +1,41 @@
...
-                <span class="icon-bar"></span>
-            </button>
-            <a class="navbar-brand" href="{{ route('home') }}">Laravel.io</a>
-        </div>
+<nav class="min-h-20 mb-8">
+    <div class="container mx-auto">
+        <div class="nav-wrapper">
+            <div class="nav-container">
+                <a class="my-4" href="{{ route('home') }}">
+                    <img src="/images/laravelio.png" alt="Laravel.io Logo" class="w-40"/>
```
Comment (driesvints) [inline resources/views/layouts/_nav.blade.php]:
```suggestion
                    <img src="{{ asset('images/laravelio.png') }}" alt="Laravel.io Logo" class="w-40"/>
```

File: resources/views/layouts/_nav.blade.php [hunk 4]
```diff
@@ -1,31 +1,41 @@
...
+                    </svg>
+                </button>
+
+
+                <button type="button" id="sidebar-close" class="hidden flex items-center lg:hidden text-gray-500 focus:outline-none">
+                    <svg class="fill-current w-4 h-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
+                        <path d="M10 8.586L2.929 1.515 1.515 2.929 8.586 10l-7.071 7.071 1.414 1.414L10 11.414l7.071 7.071 1.414-1.414L11.414 10l7.071-7.071-1.414-1.414L10 8.586z"/>
+                    </svg>
+                </button>
+
```
Comment (driesvints) [inline resources/views/layouts/_nav.blade.php]:
```suggestion
```

File: resources/views/layouts/_nav.blade.php [hunk 5]
```diff
@@ -1,31 +1,41 @@
...
+                    <img src="/images/laravelio.png" alt="Laravel.io Logo" class="w-40"/>
+                </a>
+
+                <button type="button" id="sidebar-open" class="flex items-center lg:hidden text-gray-500 focus:outline-none">
+                    <svg class="fill-current w-4 h-4" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20">
+                        <path d="M0 3h20v2H0V3zm0 6h20v2H0V9zm0 6h20v2H0v-2z"/>
+                    </svg>
+                </button>
+
+
```
Comment (driesvints) [inline resources/views/layouts/_nav.blade.php]:
```suggestion
```

File: tailwind.config.js [hunk 6]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  theme: {
+    extend: {
+      fontFamily: {
+        sans: ['Lato', 'sans-serif']
```
Comment (driesvints) [inline tailwind.config.js]: Import `const defaultTheme = require('tailwindcss/defaultTheme')` above and change this to:

```suggestion
        sans: [
          'Lato',
          ...defaultTheme.fontFamily.sans,
        ]
```

That way you have all the fallbacks from the default Tailwind config.

File: resources/css/app.css [hunk 7]
```diff
@@ -1,5 +1,118 @@
...
+    nav ul.subnav li {
+        @apply border-b-2 p-2;
+    }
+
+    nav ul.subnav li:last-child {
+        @apply border-none;
+    }
+
+    nav li.active {
+        @apply border-b-2 border-green-500
```
Comment (driesvints) [inline resources/css/app.css]:
```suggestion
        @apply border-b-2 border-green-500;
```

File: resources/css/app.css [hunk 8]
```diff
@@ -1,5 +1,118 @@
...
+        @apply mx-2;
+        padding: 1.675rem 0;
+    }
+
+    nav ul.nav>li:first-child {
+        @apply ml-0
+    }
+
+    nav ul.nav>li:last-child {
+        @apply mr-0
```
Comment (driesvints) [inline resources/css/app.css]:
```suggestion
        @apply mr-0;
```

File: resources/css/app.css [hunk 9]
```diff
@@ -1,5 +1,118 @@
...
+        @apply mb-0;
+    }
+
+    nav ul.nav>li {
+        @apply mx-2;
+        padding: 1.675rem 0;
+    }
+
+    nav ul.nav>li:first-child {
+        @apply ml-0
```
Comment (driesvints) [inline resources/css/app.css]:
```suggestion
        @apply ml-0;
```

File: resources/css/app.css [hunk 10]
```diff
@@ -1,5 +1,118 @@
...
+nav ul.nav:last-child {
+    @apply mb-4;
+}
+
+nav ul.nav li {
+    @apply py-2;
+}
+
+nav ul.nav li.active {
+    @apply text-black
```
Comment (driesvints) [inline resources/css/app.css]:
```suggestion
    @apply text-black;
```

**Comment by driesvints** [comment]
Also, I see that you used a merge commit. I usually try to merge by rebasing and the ui in Github is now blocked from doing that. It's always best that you try rebase from the base branch so you have a clean history in your PR. Could you maybe try to remove the merge commit and rebase from the next-version branch? Thanks!

Screen Shot 2019-06-27 at 22 44 00

Really awesome work!

**Comment by joedixon** [comment]
No problem, I’ll get on this tomorrow 👍

That was a fast review 🔥

**Comment by joedixon** [comment]
All updates made 👍

**Pull request merged**